README.md

```diff
@@ -128,6 +128,7 @@ Two Chinese version of this list are available [here](https://github.com/alim0x/
 - [GhettoSkype](https://github.com/stanfieldr/ghetto-skype) - open source web wrapper for Skype. [![Open-Source Software][OSS Icon]](https://github.com/stanfieldr/ghetto-skype) ![Freeware][Freeware Icon]
 - [Gitter](https://gitter.im/) - Gitter — Where developers come to talk. Gitter is designed to make community messaging, collaboration and discovery as smooth and simple as possible. [![Open-Source Software][OSS Icon]](https://github.com/gitterHQ/services) ![Freeware][Freeware Icon]
 - [HexChat](https://hexchat.github.io/) - HexChat is an IRC client based on XChat, but unlike XChat it’s completely free for both Windows and Unix-like system.[![Open-Source Software][OSS Icon]](https://github.com/hexchat) ![Freeware][Freeware Icon]
+- [Irssi](https://github.com/irssi/irssi) - The client of the future. A terminal-based IRC client. [![Open-Source Software][OSS Icon]] ![Freeware][Freeware Icon]
```

Comment:
I think you should replace the decription. This description is too general and brings no value, especially the term "The client of the future". I see on their project page the default description goes like this : " Irssi is a modular chat client that is most commonly known for its text mode user interface ". This one is better. Moreover, they list a few features, which would deserves mentioning in the description 👍